draft-demarco-oauth-nonce-endpoint.md

```diff
@@ -41,7 +41,7 @@ informative:
   BCP195: BCP195
 
 --- abstract
-This document defines the Nonce Endpoint for OAuth 2.0 implementations [RFC6749]. It details how an Authorization Server generates and issues opaque Nonces and how a client can learn about this endpoint to obtain a Nonce generated an Authorization Server or another entity.
+This document defines the Nonce Endpoint for OAuth 2.0 implementations [RFC6749]. It details how a Server generates and issues opaque Nonces and how a client can learn about this endpoint to obtain the Nonce.
```

Comment:
```suggestion
This document defines a Nonce Endpoint and details how a Server generates and issues opaque Nonces and how a client can learn about this endpoint to obtain the Nonce.
```

Reply:
if the goal is to make this endpoint normatively citable from things other than OAuth.

Reply:
@giadas ILGTM

Reply:
For me it is ok as suggested by Orie. Minor: I suggest to have both entities lowercases "Server" -> "server" (or both capitalize "client" -> "Client")